CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.5.0]
+### Breaking Changes
```

Comment:
Should the README be updated to use `values` rather than `set-values` in the Quickstart?

Also, should `values-strings` be deprecated now, since you'd just have to explicitly make something a string in your yaml? like
```yaml
values:
  bar: "2"
```

Reply:
Actually, values-strings is exposing a feature in helm where you can set `--set-string` (see [Docs Here](https://helm.sh/docs/helm/#options-43))

Reply:
Oops - didn't address your first question - yes the readme needs updating - i'll do that now. :) good catch